src/OceanSeaIceModels/freezing_limited_ocean_temperature.jl

```diff
@@ -18,7 +18,7 @@ All fluxes are shut down when the surface is below the `T < Tₘ` except for hea
 
 the melting temperature is a function of salinity and is controlled by the `liquidus`.
 """
-FreezingLimitedOceanTemperature(FT=Float64; liquidus = LinearLiquidus(FT)) = FreezingLimitedOceanTemperature(liquidus) 
+FreezingLimitedOceanTemperature(; liquidus = LinearLiquidus()) = FreezingLimitedOceanTemperature(liquidus) 
```

Comment:
```suggestion
FreezingLimitedOceanTemperature() = FreezingLimitedOceanTemperature(LinearLiquidus())
```

just my opinion that we don't need a kwarg when its just one property

Reply:
but i guess this is a new opinion

Reply:
its a new anti-kwarg Greg

Reply:
ahah okok, agreed